core/tx_pool.go

```diff
@@ -26,6 +26,8 @@ import (
 	"sync"
 	"time"
 
+	errors2 "github.com/pkg/errors"
```

Comment:
don't do this. pkg/errors is a superset of the stdlib errors, so you don't need to keep plain error and errors. just remove error and use pkg/errors without this error2 alias

Reply:
alright, changed it in latest